docs/docs/migration/60-70.md

```diff
@@ -48,10 +54,10 @@ The language of string regexes has been updated to be more in line with standard
 
 * The `RuntimeFarkle<T>` type was renamed to @"Farkle.CharParser`1".
 * The `PostProcessor<T>` type was renamed to @"Farkle.Parser.Semantics.ISemanticProvider`2" and can support other character types, for future extensibility.
-* Instead of an F# `Result`, the parser functions return values of type @"Farkle.ParserResult`1".
-* Instead of a discriminated union, parser errors have a type of `object`, and can be converted to a human-readable message by calling `ToString()`. For some specific error types, you can try casting the error object to one of the types defined in the @"Farkle.Diagnostics.Oriskany" namespace.
+* Instead of an F# `Result`, the parser functions return values of type @"Farkle.ParserResult`1". In the F# API, you can use the `|ParserSuccess|ParserError|` active pattern to match a parser result, or the `ParserResult.toResult` function to convert it to an F# `Result`.
 * The low-level tokenizer API has substantially changed, to the extent that migrating a tokenizer is tantamount to rewriting it. Due to a lack of known third-party uses, no specific guidance will be given. If you have a grammar using a custom tokenizer, you can take a look at the [tokenizer design document][tokenizer-design], or [a sample indent-based grammar][indent-based], to get an idea on how to write a custom tokenizer in Farkle 7.0.
 * The `Farkle.AST` type was removed.
+* The `Farkle.Position` type was renamed to @"Farkle.TextPosition".
```

Comment:
I didn't add an F# alias (or C# stub) to this, nor to some other types like `FarkleError` that are used by dependents. It is not a goal to make every F# project using Farkle 6 upgrade to 7 build with zero errors; the goal is for people to be able to easily navigate themselves through these errors if they exist.